### DISCUSSION TITLE
Prevent collapsing other trees on opening new tab

### DISCUSSION THREAD

**Discussion by Fak3** [piroor/treestyletab#3611] [Q&A] [answered]
TST version: 4.0.22

In previous version there was an option to keep tab tree expanded when new tab is opened. Now other treebranches always collapse when new tree appears. Is there a way to prevent that?

**Comment by irvinm** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/b4100d75-511f-403a-a05f-e9eb298629e0)

![image](https://github.com/user-attachments/assets/9176ffe4-915e-49b3-8ae0-5f6cd0fed212)

**Reply by Fak3** [reply]
Thank you!
